common/modifiers/00_activity_feast_modifiers.txt

```diff
@@ -169,6 +175,8 @@ feast_livestock_at_feast_modifier = {
 	stress_gain_mult = -0.25
 	light_cavalry_damage_mult = 0.1
 	heavy_cavalry_damage_mult = 0.1
+	light_aerial_damage_mult = 0.1
+	massive_aerial_damage_mult = 0.03
```

Comment:
Why it has bigger buff than heavy cavalry?

Reply:
0.1>0.03